frontend/app/utils/AwsAsync.scala

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import com.amazonaws.AmazonWebServiceRequest
+import com.amazonaws.handlers.AsyncHandler
+import java.util.concurrent.{Future => JFuture}
+
+import scala.concurrent.{Future, Promise}
+
+class AwsAsyncHandler[Request <: AmazonWebServiceRequest, Response] extends AsyncHandler[Request, Response] {
```

Comment:
Yup that's correct. I guess we could refactor it out into membership-common but that doesn't  really feel like a good place for it either. I think this should all sit more naturally in the new site hopefully.